### DISCUSSION TITLE
Wallpaper in Main-image config: Help pls.

### DISCUSSION THREAD

**Discussion by Crrene-p** [Godiesc/firefox-gx#121] [Q&A] [answered]
I'm new to this github stuff and I'm getting confused when I try and change the backround. I created the firefoxgx.main-image preference in the about:config page and now I'm diving into my files to change the background. After I paste my desired image into the chrome/newtab file and I rename the file to main-dark-image, it doesn't work. I try to resize the already existing wallpaper to fit the screen, but I got overwhelmed by the code in the chrome/components/ogx_root-personal. I didn't want to break anything. Will anyone help me through the process of changing my wallpaper or somehow explain the process in a simpler way?

**Comment by Godiesc** [comment]
The name of the image is main-image-dark.png , and be sure the image is in png format, that's all.

> I try to resize the already existing wallpaper to fit the screen, but I got overwhelmed by the code in the chrome/components/ogx_root-personal

The "resize" of the image is not to do in the code, you can use some software or online page to resize your image with them size of your monitor.

**Comment by Crrene-p** [comment]
Thanks for the instructions, but my smol brain must've messed up somewhere. I changed the image type to a png file and I restarted the browser, and it isn't working yet. 
![Screenshot 2024-01-10 093722](https://github.com/Godiesc/firefox-gx/assets/156001122/357e8577-ee8e-4755-b0ea-1cd2b26c85da)

**Reply by Godiesc** [reply]
jeje, I hope you follow the [instruction to apply the theme](https://github.com/Godiesc/firefox-gx#how-to-install): and after that you should put your image in the "newtab" folder than is on your firefox profile folder; in your screenshot you show the images but you are in your "downloads" folder. Give me a ScreenShot how Your Firefox looks like to know if the theme is applied.

**Comment by Crrene-p** [comment]
Ahhhh! Thank you! Also, I noticed the image I use isn't sized on the page correctly. The picture is the same size as my monitor. Is there any fix for that? 
![Screenshot 2024-01-10 114645](https://github.com/Godiesc/firefox-gx/assets/156001122/f999caf9-472e-48f6-beb1-9723fd9cfb74)

**Reply by Godiesc** [reply, marked as answer]
2 reasons could be the problem:
1.- You have the windows task-bar on left or right and that change the width of the Firefox window.
2.- You don't resize correctly the image, for example mi monitor size is this:

![Sin título](https://github.com/Godiesc/firefox-gx/assets/22057609/532f81be-85d3-4032-bf07-165c6eb15e89)

so, my image size should be that:

![imagen](https://github.com/Godiesc/firefox-gx/assets/22057609/a23913c0-d413-4402-999d-f656b2d0957e)

PD: the image will align only with Firefox maximized.
